Improve section on UserTS setup (#388)
TYPO3-Documentation/TYPO3CMS-Reference-TSconfig: Documentation/UserTsconfig/Setup.rst

    boolean

:aspect:`Description`

>     Allow file upload directly from within CE edit masks.

Reviewer: ```suggestion
    Allow file upload directly from within edit masks of content elements.
```

Maybe "forms" or "backend forms" instead of "edit masks"?

>     Caution: In most cases the uploaded file will in put into `fileadmin/user_upload/`.

Reviewer: ```suggestion
    Caution: In most cases the uploaded file will be stored in :file:`fileadmin/user_upload/`.
```

Maybe add an admonition? `.. important::` or `..note::` . Using caution admonition may be to harsh, I think.

Author: I reworked it and added cross-references, whoohoo ;-)

:aspect:`Description`

>     One of the language-keys. For current options see

Reviewer: ```suggestion
    One of the language keys. For current options see
```

    If set, then the hideAtCopy feature for records in TCE will not be used.

.. index:: File list; Show hidden files and folders

> showHiddenFilesAndFolders

Reviewer: Can you give a reference where this comes from?

Author: https://github.com/TYPO3/typo3/blob/main/typo3/sysext/core/Classes/Authentication/BackendUserAuthentication.php#L1440

    string

:aspect:`Description`

>     Name of the module that is called when the user logs into the Backend, e.g.

Reviewer: ```suggestion
    Name of the module that is called when the user logs into the backend, for example,
```

We want to avoid that abbreviation.

    positive integer

:aspect:`Description`

>     Maximum length of rendered record titles in the BE interface.

Reviewer: ```suggestion
    Maximum length of rendered record titles in the backend interface.
```

>     It's used in several places: page tree, edit masks, workspace module, etc.

Reviewer: ```suggestion
    It is used in several places: page tree, edit masks, workspace module, etc.
```

We try to avoid `'`.
"edit masks" vs. "backend forms"? What do you think?

>     Tip: to find out where exactly set it to a low number.

Reviewer: ```suggestion
    Tip: to find out where this setting is applied, set it to a low number.
```

It is better readable now, I think. I had to read it twice to understand. Maybe also use the admonition `.. tip::`?
